rules/qc/repeat_masking.rules

```diff
@@ -26,21 +26,32 @@ rule repeatmasker:
     threads:
         Cfg['qc']['threads']
     params:
-        dir = str(QC_FP/'masked'/'log')
+        dir = str(QC_FP/'masked'/'log'),
+        skip = Cfg['qc']['mask_reads'] if 'mask_reads' in Cfg['qc'].keys() else False
```

Comment:
Good catch! Maybe we should change the option name to `skip_complexity_masking`?